BasicSample/Assets/Shared/Scripts/RuntimeInfo.cs

```diff
@@ -15,18 +17,26 @@ public class RuntimeInfo : MonoBehaviour
         [SerializeField]
         private TextMesh runtimeText = null;
 
+        [SerializeField]
+        private ARSession arSession = null;
+
         private void Update()
         {
             if (m_frameCountSinceLastUpdate-- <= 0)
             {
                 m_frameCountSinceLastUpdate = m_frameCountToUpdateFrame;
+                arSession = FindObjectOfType<ARSession>();
```

Comment:
sorry i just notice this.  The "find object of type" code should be moved to "Start" function which only need to happen once instead of every frame.

Reply:
Done